feature/platform/tests/telemetry_inventory_test/README.md

```diff
@@ -31,6 +31,7 @@ card, power supply, disk, flash, NPU, transceiver, fabric card), validate:
 *   /components/component/state/id
 *   /components/component/state/mfg-date
 *   /components/component/state/mfg-name
+*   /components/component/state/model-name
```

Comment:
Please refactor the list of paths to match the yaml format  in the [README template](https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#openconfig-path-and-rpc-coverage).